pkg/migrations/sql/versions/dev/0.1.5-dev/1-update_execute_everywhere.sql

```diff
@@ -0,0 +1,30 @@
+--table to save commands so they can be run when adding new nodes
+ CREATE TABLE SCHEMA_CATALOG.remote_commands(
+    key TEXT PRIMARY KEY,
+    seq SERIAL,
+    transactional BOOLEAN,
+    command TEXT
+);
```

Comment:
Do we have to pre-populate this table with the non-idempotent stuff? I don't see how it'll end up in the table otherwise...

Reply:
ahh, I think that's [below](https://github.com/timescale/promscale/pull/423/files#diff-e992ae989c80b1f77be0a5f601764a61035072e169adbc8f931f77d3c8397b0cR22-R116l), and just wasn't in the commit I was reviewing